### PR TITLE
Setting default tab page to about:newTab. Fixes: #184

### PR DESCRIPTION
**Pull request by jonathanKingston**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -504,9 +505,9 @@ const ContainerService = {
         userContextId = args.userContextId;
       }
 
-      const tab = browserWin.gBrowser.addTab(args.url || null, { userContextId });
+      const tab = browserWin.gBrowser.addTab(args.url || DEFAULT_TAB, { userContextId });
       browserWin.gBrowser.selectedTab = tab;
-      browserWin.focusAndSelectUrlBar();
+      browserWin.focusUrlBar();
```
Comment (groovecoder) [inline index.js]: We decided to stop selecting the url bar? 😢

Comment (bakulf) [inline index.js]: I prefer browserWin.focusAndSelectUrlBar(), plus it's not related with this PR. Separate commit? separate PR?

Comment (jonathanKingston) [inline index.js]: Wait why would you want that? Anyway I did this as with the `newTab` it showed and was selected.

Comment (jonathanKingston) [inline index.js]: Pushed fix anyway.

**Pull request merged**